### DISCUSSION TITLE
A basic question

### DISCUSSION THREAD

**Discussion by nk8020** [uBlockOrigin/uBlock-issues#2107] [Q&A] [answered]
Tried to follow the instructions for blocking 3rd party frames and script global, but enabling some local.

1st -  the three bars (stoplight type red-2-green) are absent in my 1.42.4 version - even though i checked the "advanced user" box, I get 
         only a grey-out field for allowing specific sites/ domains both for global policy and local (either red or greyed-out, or just white (no
         exception to the global rule). --How do I implement it and enable specific websites?
2nd - Is there another way to create exceptions to sites/domains, temporarily/permanently with a push of a button? -I was under the
          impression that there was such an option, but could not find how to implement it.

-Any help in resolving this issues for me will be greatly appreciated, even if it does seem basic.

**Comment by gothic-bum** [comment, marked as answer]
> 1st - the three bars (stoplight type red-2-green) are absent in my 1.42.4 version - even though i checked the "advanced user" box, I get
only a grey-out field for allowing specific sites/ domains both for global policy and local (either red or greyed-out, or just white (no
exception to the global rule). --How do I implement it and enable specific websites?  
  
https://github.com/gorhill/uBlock/wiki/Dynamic-filtering:-quick-guide#allow-rules

> 2nd - Is there another way to create exceptions to sites/domains, temporarily/permanently with a push of a button? -I was under the
impression that there was such an option, but could not find how to implement it.  
  
https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-large-power-button

**Comment by nk8020** [comment]
Thank you kindly gothic-bum.
This time I somehow was able to get more of the explanation.
